### PR TITLE
add the timestamp to the micro texts

### PR DESCRIPTION
**Pull request by dudethatbe**
image

### CODE REVIEW

File: app/components/add-datafruit.js [hunk 1]
```diff
@@ -19,6 +19,10 @@ export default class AddDatafruit extends Component {
   @tracked
   datafruit = null;
 
+  get currentTimestamp() {
+    let timestamp = new Date(this.currentDatafruit.createdAt);
```
Comment (mcfiredrill) [inline app/components/add-datafruit.js:23]: the issue seems to be that `currentDatafruit` might be `undefined` here

Comment (dudethatbe) [reply app/components/add-datafruit.js]: thanks! tests are passing now. I was wondering if it should return anything if there isn't a microtext yet 🤔

Comment (dudethatbe) [reply app/components/add-datafruit.js]: oops that causes a lint error anyways. posted a fix

**Comment by mcfiredrill** [comment]
Maybe just the date without the time is fine? idk

**Review by mcfiredrill — APPROVED** [review]
LGTM but maybe just the date without the time is good enough

**Pull request merged**